api/storage_service.cc

```diff
@@ -548,6 +563,10 @@ static future<json::json_return_type> describe_ring_as_json(sharded<service::sto
     co_return json::json_return_type(stream_range_as_array(co_await ss.local().describe_ring(keyspace), token_range_endpoints_to_json));
 }
 
+static future<json::json_return_type> describe_ring_as_json_for_table(sharded<service::storage_service>& ss, keyspace_and_table_name ks_table) {
```

Comment:
```suggestion
static future<json::json_return_type> describe_ring_as_json_for_table(const sharded<service::storage_service>& ss, keyspace_and_table_name ks_table) {
```

now that `describe_ring_for_table()` already uses the `const` specifier, let's add more constness here as well.